test/selenium/signInPage.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+var SignedInPageView = require('./lib/view/signed_in/view.js');
+var signedInPageView;
+
+function SignInPage(driver) {
```

Comment:
Initially I defined a SetUpPage for the screen where you have to insert psw and confirm, but changed to this SignIn because is the initial screen we are on in this test following the flow of the previous ones...the name would not match with the view shown if I leave it as SetUpPage...that is why I changed the name to SingedIn...